todo-app/src/App.js

```diff
@@ -1,13 +1,39 @@
 import React, { Component } from 'react';
+import Task from './Task' 
+import todoItems from './todo.json';
 
-class App extends Component {
+
+export default class App extends Component {
+    state = {
+    todos: todoItems
+  }
+
+handleTaskDone = (todoID)=>{
```

Comment:
The name `handleTaskDone` is not accurate, because it marks todos as done _and also_ as not done. A more accurate name would be something like `handleToggleTodoStatus`, or something.

Also, be careful with your formatting! This line should be indented, and there should be spaces around your `=>`.